hw2/03-form.html

```diff
@@ -12,8 +13,35 @@
     <script defer src="03-form.js"></script>
   </head>
   <body class="bg-dark">
-    <form class="bg-light border rounded w-50 mx-auto mt-5 p-3">
+    <form class="bg-light border rounded w-25 mx-auto mt-5 p-3">
       <h1 class="mt-2 mb-4">Contact Form</h1>
+      <div class="form-group">
+        <label for="inputemail">Name * </label>
```

Comment:
Name, email, feedback, and newsletter text could use a little top and bottom margins to match the spacing in the homework document.